components/docs-chef-io/content/automate/ha_backup_restore_aws_s3.md

```diff
@@ -158,3 +158,41 @@ While running the restore command, If it prompts any error follow the steps give
 -  Also check the hab svc status in automate node by running `hab svc status`.
 -  If the deployment services is not healthy then reload it using `hab svc load chef/deployment-service`.
 -  Now, check the status of Automate node and then try running the restore command from bastion.
+
+For Disaster Recovery or AMI upgarde, while running the restore in secondary cluster which is in different region follow the steps given below.
+
+-  First, Take the bakup in Secondary Cluster
+-  Then make a curl request `curl -XGET https://localhost:9200/_snapshot?pretty -u admin:admin -k`
```

Comment:
This CURL is only supported in the OpenSearch node. Make sure to mention that, or add the endpoint to make same request from Automate Node

Reply:
Done